### PR TITLE
clean EVALB via make, add to Dockerfile

### PR DESCRIPTION
**Pull request by DeNeutoy**
FYI @cristipp

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -65,6 +65,8 @@ COPY setup.py setup.py
 ARG CACHE_MODELS=false
 RUN ./scripts/cache_models.py
 
+RUN cd scripts/EVALB && make && cd ..
```
Comment (matt-gardner) [inline Dockerfile]: How long does this take?  Is it possible to move it earlier in the docker file, so it doesn't get rebuilt every time `allennlp/` changes?

Comment (joelgrus) [reply Dockerfile]: it looks like on teamcity it only took 1 second:

http://build.allennlp.org/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=3034&_focus=190

Comment (matt-gardner) [reply Dockerfile]: Plus a bunch of annoying output...  I'm just thinking of when I use `run_with_beaker.py`, with changes to `allennlp`.  Maybe we can move the `scripts/` copy command to be the first copy, then have this command, then the rest of the copies?

Comment (DeNeutoy) [reply Dockerfile]: good point, i'll change that because it is a little alarming

Comment (matt-gardner) [reply Dockerfile]: Alternatively, you could just pipe the output to /dev/null, or something.  Or do both of those.

**Pull request merged**